.github/workflows/template.workflow-init.yml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Workflow
+
+name: "Apache Flink Workflow Initialization"
+
+on:
+  workflow_call:
+    outputs:
+      runner_config:
+        description: "The runs-on configuration that can be used in the runs-on parameter."
+        value: ${{ jobs.workflow_init.outputs.runner_config }}
+
+permissions: read-all
+
+jobs:
+  workflow_init:
+    name: "Initialize Workflow"
+    # no need to fix a specific ubuntu version here
```

Comment:
should we have some more clarifying comments where version should be changed in case of upgrade?
May be extract it into a dedicated var?

Reply:
hm, not sure how we should document that. Generally, we're using fixed VM labels (i.e. `ubuntu-22.04` right now) in all other locations. This is a location where I didn't do that. 

If we decide to upgrade the VM image, the most natural way (IMHO) would be to look for the term `ubuntu-22.04`. I added the comment here to clarify that we're not requiring a fixed version here (to have less maintenance effort) because we're not really rely on any dependency (except for some basic shell scripts). 

I will add more reasoning to the comment, though, to clarify why we don't need the fixed version here :+1: